keyboards/handwired/scottokeebs/scotto34/keymaps/default/config.h

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define options
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
```suggestion
```
This behaviour will be a default when #20211 is merged with `master` very soon. Its presence in your keymaps will trigger build errors.